mithril-aggregator/src/certifier_service.rs

```diff
@@ -673,4 +721,56 @@ mod tests {
             .unwrap();
         assert!(create_certificate_result.is_none());
     }
+
+    #[tokio::test]
+    async fn test_epoch_gap_certificate_chain() {
+        let builder = MithrilFixtureBuilder::default();
+        let certifier_service = setup_certifier_service(&builder.build(), &[]).await;
+        let certificate = fake_data::genesis_certificate("whatever");
+        let epoch = certificate.beacon.epoch.offset_by(2).unwrap();
```

Comment:
You can just use an addition here: `let epoch = certificate.beacon.epoch + 2;` :)